pkg/apis/noobaa/v1alpha1/noobaa_types.go

```diff
@@ -539,3 +560,14 @@ const (
 	// AutoscalerTypeHPAV2 is hpav2
 	AutoscalerTypeHPAV2 AutoscalerTypes = "hpav2"
 )
+
+type BucketLoggingTypes string
```

Comment:
Add a comment explaining what this type is used for

Reply:
Added, Thanks.